py/desitarget/sv1/sv1_cuts.py

```diff
@@ -1770,10 +1765,10 @@ def set_target_bits(photsys_north, photsys_south, obs_rflux,
     desi_target |= qsoz5_north * desi_mask.QSO_Z5_NORTH
 
     # ADM combined per-bit information for the LRGs...
-    desi_target |= lrginit_4 * desi_mask.LRG_INIT_4PASS
-    desi_target |= lrgsup_4 * desi_mask.LRG_SUPER_4PASS
-    desi_target |= lrginit_8 * desi_mask.LRG_INIT_8PASS
-    desi_target |= lrgsup_8 * desi_mask.LRG_SUPER_8PASS
+    desi_target |= lrg_opt * desi_mask.LRG_OPT
+    desi_target |= lrg_ir * desi_mask.LRG_IR
+    desi_target |= lrg_sv_opt * desi_mask.LRG_OSV_PT
```

Comment:
In the `py/desitarget/sv1/data/sv1_targetmask.yaml` file, you called this bit `LRG_SV_OPT` (not `LRG_OSV_PT`). I'm not sure why this isn't failing in your local tests, but unit tests caught it.

Reply:
Thanks. It's fixed.